index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
It's smart to use the HTML template starter on IDE's. It gives us a template HTML to start with some metadata and helpful configs. If you are using VS Code, try typing the character ! in an empty HTML file and click the first option.

Reply:
I made a copy of my index.html file, erased the content in the copied version and then tried <! > but nothing came up. Perhaps I am misunderstanding the direction being given in the comment above. I can schedule a meeting for us to go over this and the other comments too.